apps/challenges/aws_utils.py

```diff
@@ -305,6 +305,214 @@
 }}
 """
 
+task_definition_static_code_upload_worker = """
```

Comment:
@savish28 why do we need a new task definition for `static_code_upload_worker`? The task definition will be same as `code_upload_worker` task definition. If you need additional environment variables we can add it to `code_upload_worker` task def.

Reply:
@Ram81 It is not just the change in env variables but also the container definitions. The static code has two containers and code upload has only one so needed to create a new defn